app/views/mailers/digest_mailer/digest_email.html.erb

```diff
@@ -21,12 +21,12 @@
   <em>
     <% if @user.experience_level == nil %>
       <b>
-        You can now add your <%= SiteConfig.community_action %> experience level to your account in order to receive more relevant content.
+        You can now add your experience level to your account in order to receive more relevant content.
```

Comment:
Opted to just remove `community_action` here and not replace it with anything since I think "experience level" is self-explanatory (it exists as a field on a user's profile).